src/platform/silabs/multi-ota/OTAMultiImageProcessorImpl.cpp

```diff
@@ -20,23 +20,26 @@
 #include <app/clusters/ota-requestor/OTARequestorInterface.h>
 #include <lib/support/BufferReader.h>
 #include <platform/DiagnosticDataProvider.h>
-#include <platform/internal/CHIPDeviceLayerInternal.h>
-#include <platform/internal/GenericConfigurationManagerImpl.h>
-
 #include <platform/silabs/multi-ota/OTAMultiImageProcessorImpl.h>
 
 using namespace chip::DeviceLayer;
 using namespace ::chip::DeviceLayer::Internal;
 
 static chip::OTAMultiImageProcessorImpl gImageProcessor;
 
-#if SL_WIFI
+#if SL_WIFI && !SLI_SI91X_MCU_INTERFACE
 #include <platform/silabs/wifi/wf200/platform/spi_multiplex.h>
 #endif // SL_WIFI
 
 extern "C" {
+#ifdef SLI_SI91X_MCU_INTERFACE
+#include "sl_si91x_driver.h"
+#include "sl_si91x_hal_soc_soft_reset.h"
+#include <platform/silabs/platformAbstraction/SilabsPlatform.h>
+#else // This is not needed for the 917 SoC; it is required for EFR host applications
 #include "btl_interface.h"
-#include "sl_core.h"
+#include "em_bus.h" // For CORE_CRITICAL_SECTION
```

Comment:
Wait are we sure about this em_bus replacing sl_core.h ? This seems wrong. sl_core is newer than em_bus and combines series2 and series3. The standard ota_ProcessorImpl uses sl_core.h and uses critical sections for s2

Reply:
Ok I dont have context on this change and I think Shrinivas has added this long ago. 
Updated with `sl_core.h`